src/test/kotlin/src/step2/CalculatorTest.kt

```diff
@@ -0,0 +1,30 @@
+package src.step2
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+
+class CalculatorTest {
+
+    @Test
+    fun `연산자가 없을때`() {
+        Assertions.assertThatExceptionOfType(IllegalArgumentException::class.java)
```

Comment:
```suggestion
        assertThrows<IllegalArgumentException> { StringCalculator.execute("1234") }
```

assertThrows를 사용하면, 예외 처리에 대해서도 간결하게 테스트 코드를 짤 수가 있습니다.

assertThrows와 assertThatExceptionOfType의 차이점은 무엇일까요? 🤔

Reply:
assertThatThrownBy는 Junit4의 기능이고 assertThrows는 Junit5의 기능입니다. assertThrows를 사용하면 더 간결한 예외 테스트 코드를 작성할 수 있습니다.